### ISSUE TITLE
parse error message could be way better

### ISSUE THREAD

**Issue by nicks** [facebook/buck#466]
Create a BUCK file:

```
android_prebuilt_aar(
   name = 'foo',
   binary_jar = 'foo.aar')
```

Run `buck build :foo`

buck version 2a4dd219a5b0081c67f0512c70305986ca31a301

Expected:
some error message that says "binary_jar" is not an argument to "android_prebuilt_aar"

Actual:
`BUILD FAILED: Parse error for build file /Users/nick/git/donkey/BUCK: End of input at line 1 column 1`
which doesn't even have reasonable line number or column info.

**Comment by Coneko** [comment]
We've fixed this, the fix will be out in the next sync. I think you can get the error message by passing a different verbosity level with the `-v` option.

**Comment by Coneko** [comment]
Fixed in ef5cafc5cec1f9d96081b4a36e3ef61fff24892d.

**Comment by nicks** [comment]
thanks!!
